assets/js/components/block-title/editor.scss

```
@@ -0,0 +1,6 @@
+// Ensure textarea bg color is transparent for block titles.
+// Some themes (e.g. Twenty Twenty) set a non-white background for the editor, adn Gutenberg sets white background for text inputs, creating this issue.
+// https://github.com/woocommerce/woocommerce-gutenberg-products-block/issues/1204
+textarea.wc-block-component-title {
```

I also wonder if we need the `textarea` tag selector here. I think we should drop it in order to keep specificity low.
```suggestion
.wc-block-component-title {
```